src/activities/content/ContentFileEntity.js

```diff
@@ -120,6 +120,28 @@ export class ContentFileEntity extends ContentWorkingCopyEntity {
 		this.dispose();
 	}
 
+	/**
+	 * Replaces the file associated with the activity
+	 * @param {string} fileId unique identifier for the chosen file
+	 * @param {string} fileSystemType identifier for determining how the file is stored
+	 */
+	async replaceFile(fileId, fileSystemType) {
+		if (!this._entity || !fileId || !fileSystemType) {
```

Comment:
This is something I'm more curious about how to deal with than concerned about, but is zero a valid `fileId`?
The comment says it's a string, so "0" would be allowed, but 0 as a number would not, no?